trainer/derrian_backend/sd_scripts/library/edm2_loss.py

```
@@ -0,0 +1,167 @@
+        else:
+            weights_sd = torch.load(file, map_location="cpu")
+
+        info = self.load_state_dict(weights_sd, False)
+        return info
+    
+def create_weight_MLP(noise_scheduler: DDPMScheduler, 
+                    logvar_channels: int = 128, 
+                    lambda_weights: torch.tensor = None, 
+                    optimizer: torch.optim.Optimizer = torch.optim.AdamW, 
+                    lr: float = 2e-2,
+                    optimizer_args: dict = {'weight_decay': 0, 'betas': (0.9,0.99)},
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)